raycicmd/converter_test.go

```diff
@@ -0,0 +1,109 @@
+package raycicmd
+
+import (
+	"encoding/json"
+	"testing"
+
+	"reflect"
+)
+
+func TestConvertPipelineStep(t *testing.T) {
+	c := newConverter(&config{
+		ArtifactsBucket: "artifacts_bucket",
+		CITemp:          "s3://ci-temp/",
+
+		RunnerQueues: map[string]string{"default": "runner"},
+		Dockerless:   true,
+	}, "buildid")
+
+	for _, test := range []struct {
+		in  map[string]any
+		out map[string]any // buildkite pipeline step
+	}{{
+		in: map[string]any{"commands": []string{"echo 1", "echo 2"}},
```

Comment:
maybe also test the job_env?

Reply:
added.

was not testing docker container previously.. too lazy..